Add HumanWorker, OTTO Robots, RGL and UR Robots robotic Gems (#2517)
o3de/o3de.org: content/docs/user-guide/gems/reference/robotics/ur-robots.md

---
linkTitle: UR Robots
title: UR Robots Gem
description: The UR Robots Gem provides a collection of robotic arms assets that can be used in robotic simulations with Robot Operating System (ROS) 2 in Open 3D Engine (O3DE).
toc: true
---

<!-- # UR Robots Gem -->

The **UR Robots Gem** is an assets only Gem that provides a collection of robotic arms assets that can be used in robotic simulations with [ROS&nbsp;2 Gem](./ros2.md) in **Open 3D Engine (O3DE)**. In particular, this Gem contains the simplified models of the following robots developed by [Universal Robots](https://www.universal-robots.com/):
- [UR10](https://www.universal-robots.com/products/ur10-robot/)
- [UR20](https://www.universal-robots.com/products/ur20-robot/)

Additionally, it contains the assets of the pneumatic vacuum end effector.

> An example of its use can be found in [ROSCon2023Demo](https://github.com/RobotecAI/ROSCon2023Demo) project. The Gem is available from our [Github repository](https://github.com/RobotecAI/o3de-ur-robots-gem/). For more information, refer to the _README_ file of the Gem.

Reviewer: ```suggestion
An example of its use can be found in [ROSCon2023Demo](https://github.com/RobotecAI/ROSCon2023Demo) project. The Gem has a [Github repository](https://github.com/RobotecAI/o3de-ur-robots-gem/). For more information, refer to the _README_ file of the Gem.
```
